app/models/katello/glue/pulp/consumer.rb

```diff
@@ -197,7 +197,7 @@ def katello_agent_installed?
       def fetch_package_profile
         Katello.pulp_server.extensions.consumer.retrieve_profile(uuid, 'rpm')
       rescue RestClient::ResourceNotFound => e
-        Rails.logger.info "Failed to find profile for #{uuid}: #{e}}"
+        Rails.logger.error "Failed to find profile for #{uuid}: #{e}, #{e.backtrace.join("\n")}"
```

Comment:
I think info makes sense to me -- fetching the profile and getting a 404 isn't really an 'error' so much as it's just missing data.